themes/classic/_dev/js/selectors.js

```diff
@@ -54,7 +54,7 @@ prestashop.themeSelectors = {
     searchLink: '.js-search-link',
   },
   order: {
-    returnForm: '#order-return-form, .js-order-return-form',
+    returnForm: '#order-return-form',
```

Comment:
This will lead to a BC break and themes only using the prefixed `js-` class won't work anymore

These `js-` classes are the only thing we need but we kept old classes for retrocompatibility

```suggestion
    returnFormHeadCheckboxes: '#order-return-form table thead input[type=checkbox], .js-order-return-form table thead input[type=checkbox]',
    returnFormContentCheckboxes: '#order-return-form table thead input[type=checkbox], .js-order-return-form table thead input[type=checkbox]',
```

And then update `customer.js` accordingly like

```js
  $(`${prestashop.themeSelectors.order.returnFormHeadCheckboxes}`).on('click', function () {
    const checked = $(this).prop('checked');
    $(`${prestashop.themeSelectors.order.returnFormContentCheckboxes}`).each((_, checkbox) => {
      $(checkbox).prop('checked', checked);
    });
  });
```